src/components/code-snippet/_mixins.scss

```diff
@@ -1,5 +1,5 @@
 @mixin bx--snippet {
-  font-family: monospace;
+  font-family: 'ibm-plex-mono', monospace;
```

Comment:
Can you change the fallbacks to `'ibm-plex-mono', 'Menlo', 'DejaVu Sans Mono', 'Bitstream Vera Sans Mono', Courier, monospace`?

Reply:
That's what is in the IBM Type repo

Reply:
`DeJaVu SaNs MoNo`